keras/backend/cntk_backend.py

```diff
@@ -1512,9 +1532,11 @@ def __init__(self, inputs, outputs, updates=[], **kwargs):
                     p_list.append(grad_parameter_dict[g])
                     u_list.append(g)
                 else:
-                    raise ValueError('cntk backend: '
-                                     'found gradient not related with '
-                                     'parameters when construct trainer.')
+                    raise ValueError('cntk backend: when construct trainer,'
+                                     'found gradient node \'%s\' which is not'
```

Comment:
need space before `found`, and generally at the end of every line in this message